src/pages/Show.js

```diff
@@ -0,0 +1,59 @@
+import NavButton from "../components/NavButton"
+import { useParams, useNavigate } from "react-router-dom"
+
+const Show = ({ apartments, deleteApartment, user }) => {
+	const navigate = useNavigate()
+	const { id } = useParams()
+	console.log("apartments:", apartments)
+	const apartment = apartments.find((item) => item.id === +id)
+	console.log("apartment:", apartment)
+	// const handleDeleteApartment = () => {
+	//   deleteApartment(apartment.id)
+	//   navigate("/apartments")
+	// }
```

Comment:
There are some leftover logs and delete functionality in this component that can be removed for this PR.